plugins/google_analytics/girder_google_analytics/settings.py

```diff
@@ -0,0 +1,17 @@
+from girder.exceptions import ValidationException
+from girder.utility import setting_utilities
+
+
+class PluginSettings(object):
+    TRACKING_ID = 'google_analytics.tracking_id'
+
+
+@setting_utilities.default(PluginSettings.TRACKING_ID)
+def _defaultTrackingId():
+    return ''
```

Comment:
For static values, should we use a function? I guess this is attempting to define a normative pattern, but an alternative would be:

```python
SettingDefault.defaults[PluginSettings.TRACKING_ID] = ''
```

Also, the default here was changed from `None` to `''`, is there a reason for that?

Reply:
My intention was to avoid having plugin load mutate `SettingDefault.defaults`, and move that towards holding core setting defaults only.

As for the typing, it's a partial attempt to move towards trying to have stricter type validations for settings. In this instance, I should also update `_validateTrackingId` to require a string type.

Reply:
> As for the typing, it's a partial attempt to move towards trying to have stricter type validations for settings. In this instance, I should also update `_validateTrackingId` to require a string type.

Actually, I'll try to deal with this more comprehensively in another PR.